trulens_eval/examples/snowflake_demo/preload_apps.ipynb

```diff
@@ -24,8 +24,17 @@
     "\n",
     "load_dotenv()\n",
     "\n",
-    "tru = Tru()\n",
-    "tru.run_dashboard()"
+    "db_url = 'snowflake://{user}:{password}@{account}/{dbname}/{schema}?warehouse={warehouse}&role={role}'.format(\n",
+    "        user=os.environ['SF_USER'],\n",
```

Comment:
I think we're changing ot use `SNOWFLAKE_*` as per @coreyhu 's change

Reply:
updated